page-objects/contactUsPage.ts

```diff
@@ -0,0 +1,30 @@
+import { Page } from "@playwright/test";
+
+export class contactUsPage {
+    readonly page: Page
+    constructor(page: Page) {
+        this.page = page
+    }
+    // Creating the elements and finding locators of the mentioned ones.
+    async clickOnContactUsForm() {
+        return this.page.frameLocator('#framelive').locator('#contact-link').getByRole('link', { name: 'Contact us' }).first().click()
+    }
+    async clickOnSubjectFromDropDown() {
+        return this.page.frameLocator('#framelive').getByLabel('Subject').click()
+    }
+    async clickOnWebmasterInDropDown() {
+        return this.page.frameLocator('#framelive').getByLabel('Subject').selectOption('Webmaster')
+    }
+    async emailInput() {
+        return this.page.frameLocator('#framelive').getByPlaceholder('your@email.com')
+    }
+    async subjectMessage() {
+        return this.page.frameLocator('#framelive').getByPlaceholder('How can we help?').fill('Do you have any new upcoming products?')
+    }
+    async sendButton() {
```

Comment:
If you have  to click on something, please explain that in the name. Please add '()' on click function